src/components/flyout/flyout_body.tsx

```diff
@@ -31,9 +33,21 @@ export const EuiFlyoutBody: EuiFlyoutBodyProps = ({
     'euiFlyoutBody__overflow--hasBanner': banner,
   });
 
+  const euiTheme = useEuiTheme();
+  const styles = euiFlyoutBodyStyles(euiTheme);
+
+  const cssStyles = [
+    styles.euiFlyoutBody,
+    banner && styles.euiFlyoutBody__banner,
+  ];
```

Comment:
This isn't getting applied to the correct place. We need a separate var for `euiFlyoutBody__banner`, otherwise this CSS will unintentionally affect **all** `EuiCallOut`s inside `EuiFlyout`, and we only specifically want to target EuiCallOuts inside the banner DOM
```suggestion
    const cssStyles = [styles.euiFlyoutBody];
```